agent/agent-tooling/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java

```diff
@@ -200,16 +203,18 @@ public AzureMonitorExporterBuilder clientOptions(ClientOptions clientOptions) {
    * @throws IllegalArgumentException If the connection string is invalid.
    */
   public AzureMonitorExporterBuilder connectionString(String connectionString) {
+    // TODO (trask) share connection string parsing code
     Map<String, String> keyValues = extractKeyValuesFromConnectionString(connectionString);
     if (!keyValues.containsKey("InstrumentationKey")) {
       throw logger.logExceptionAsError(
           new IllegalArgumentException("InstrumentationKey not found in connectionString"));
     }
     this.instrumentationKey = keyValues.get("InstrumentationKey");
     String endpoint = keyValues.get("IngestionEndpoint");
-    if (endpoint != null) {
-      this.endpoint(endpoint);
+    if (endpoint == null) {
+      endpoint = "https://dc.services.visualstudio.com";
```

Comment:
can we use DefaultEndpoints.INGESTION_ENDPOINT?

Reply:
I'm working on sharing ConnectionString parsing in next PR(s), this code will be gone shortly 👍